core/src/main/kotlin/org/evomaster/core/search/gene/sql/SqlMultiRangeGene.kt

```diff
@@ -0,0 +1,99 @@
+package org.evomaster.core.search.gene.sql
+
+import org.evomaster.core.logging.LoggingUtil
+import org.evomaster.core.output.OutputFormat
+import org.evomaster.core.search.gene.*
+import org.evomaster.core.search.gene.sql.SqlStrings.removeEnclosedQuotationMarks
+import org.evomaster.core.search.service.AdaptiveParameterControl
+import org.evomaster.core.search.service.Randomness
+import org.evomaster.core.search.service.mutator.genemutation.AdditionalGeneMutationInfo
+import org.evomaster.core.search.service.mutator.genemutation.SubsetGeneSelectionStrategy
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+class SqlMultiRangeGene<T>(
+        name: String,
+        val template: SqlRangeGene<T>,
+        val rangeGenes: ArrayGene<SqlRangeGene<T>> = ArrayGene(name, template)
+) : Gene(name, mutableListOf(rangeGenes)) where T : ComparableGene {
+
+    companion object {
+        val log: Logger = LoggerFactory.getLogger(SqlMultiRangeGene::class.java)
+    }
+
+    override fun getChildren(): MutableList<Gene> = mutableListOf(rangeGenes)
```

Comment:
why returning a mutable list instead of `List`?

Reply:
My bad. I fixed it.